src/module/actor/mixins/actor-conditions.js

```diff
@@ -21,21 +33,38 @@ export const ActorConditionsMixin = (superclass) => class extends superclass {
         return conditionItem;
     }
 
-    async setCondition(conditionName, enabled) {
+    /**
+     * Updates the Actor's conditions. Either adds or removes a condition Item as necessary to match the enabled argument.
+     * @param {String} conditionName The name of the condition. Must match any key from config.js SFRPG.statusEffectIconMapping. Case insensitive.
```

Comment:
The conditionName is suggested to be case insensitive in your comments, but it is case sensitive.

Reply:
Same as above